packages/@aws-cdk/aws-efs/lib/access-point.ts

```diff
@@ -139,6 +139,11 @@ export class AccessPoint extends Resource implements IAccessPoint {
    */
   public readonly accessPointId: string;
 
+  /**
+   * The filesystem of the access point
+   */
+  public readonly filesystem: IFileSystem;
```

Comment:
To be coherent with `IFileSystem`?
```suggestion
  public readonly fileSystem: IFileSystem;
```